conda_forge_tick/make_graph.py

```diff
@@ -338,6 +338,29 @@ def update_nodes_with_bot_rerun(gx):
                     )
 
 
+
+def update_nodes_with_new_versions(gx):
```

Comment:
Are you sure you want to put this into this PR?

Reply:
Just committing the "removing" will be enough ?

Reply:
Yes you can add a commit removing it. I would make certain to keep it somewhere though (maybe in a different branch?). The code looks good and we'll need it for the next phase.

Reply:
Done! (I saved in a file for later usage)